error-handlers.js

```diff
@@ -1,5 +1,5 @@
 exports.psqlErrorHandler = (err, req, res, next) => {
-    if (err.code === '22P02') {
+    if (err.code === '22P02' || err.code === '23502' || err.code === '23503') {
```

Comment:
Nice as these begin to grow you could add them to an array and do an errorCodes.includes